api_tests/preprint_providers/views/test_preprint_provider_subjects_list.py

```diff
@@ -6,6 +6,28 @@
 from osf_tests.factories import SubjectFactory, PreprintProviderFactory
 
 
+class TestPreprintProviderExists(ApiTestCase):
+
+    # Regression for https://openscience.atlassian.net/browse/OSF-7621
+
+    def setUp(self):
+        super(TestPreprintProviderExists, self).setUp()
+        self.url = '/{}/preprint_proiders/fake/'.format(API_BASE)
```

Comment:
Typo (`preprint_proiders`) which will always 404, can you fix the typo and add a passing test for sanity sake?